tests/responses/get_issue.json

```diff
@@ -72,17 +72,11 @@
       "$type": "SimpleIssueCustomField"
     },
     {
-      "value": "100-003-675",
+      "value": "1623396729",
       "name": "Multipass",
       "id": "145-36",
       "$type": "SimpleIssueCustomField"
     },
-    {
-      "value": 4003,
```

Comment:
I wouldn't now remove this test, because if I understand it correctly, now you document the following:

If integer comes back, it's always a datetime. And even if for now we don't parse the int that is actually not a datetime correctly at the moment, I would like to have it documented with a todo, instead of just removing.

Reply:
My idea was to leave the task to someone else for revision. But you didn't get me :( Ok, see updated code.

Reply:
Well, you can't leave it to me, right? So the only other candidate was @IamAbbey - which would have surely worked, but you just had to be more explicit: like I did this & that, and this part is still to be done, @IamAbbey please take care of this ASAP...

Reply:
> you just had to be more explicit

Sure, next time I will.